source/gui/settingsDialogs.py

```diff
@@ -4428,6 +4428,15 @@ def makeSettings(self, settingsSizer):
 		)
 		self.bindHelpEvent("BrailleSettingsInterruptSpeech", self.brailleInterruptSpeechCombo)
 
+		self.formattingDisplayCombo = sHelper.addLabeledControl(
```

Comment:
Does "Formatting display" option apply when "Braille mode" is set to "display speech output"? If not, i.e. if it only applies to "follow cursors" (default) mode, you should:
- Put this option after "Show selection" rether than at the end so that options that apply to "follow cursors" mode are grouped in this panel.
- disable it if "Braille mode" is not set to "Follow cursors"